app/pages/project/classify.cjsx

```diff
@@ -230,7 +230,7 @@ module.exports = React.createClass
 
   saveClassificationAndLoadAnotherSubject: ->
     @saveClassification()
-      .then @loadAnotherSubject()
+    @loadAnotherSubject()
 
```

Comment:
Would passing `@state.classification` as an argument to `@saveClassification()` be safer? Then, we don't have to worry about `@loadAnotherSubject()` executing before `@saveClassication()` can [make a copy of the classification](https://github.com/zooniverse/Panoptes-Front-End/pull/3681/files#diff-83addd0f5535ee9ffe878b365f61712bL241).